examples/iris/iris.rb

```diff
@@ -31,10 +31,10 @@ def fwd(x)
 optimizer.setup(model)
 
 iris = Datasets::Iris.new
-x = iris.each.map {|r| r.each.to_a[0..3]}
+x = iris.to_table.fetch_values(:sepal_length, :sepal_width, :petal_length, :petal_width).transpose
```

Comment:
We still need `iris_table = iris.to_table` with `Table#fetch_values` because parsed values are cached in each `Table` object. `to_table` always returns a new `Table` object.